root/components/FormRow.js

```diff
@@ -19,7 +19,10 @@ const FormRow = ({
   hasNoLabel,
   ...props
 }: Props): React.Element<'div'> => (
-  <div className={'row' + (hasNoLabel ? ' no-label' : '')} {...props}>
+  <div
+    className={'row' + (hasNoLabel /*:: === true */ ? ' no-label' : '')}
+    {...props}
+  >
```

Comment:
In cases where we have an optional property like here, I think it's better for clarity to just set the default value `hasNoLabel = false,` in the destructuring above.

Reply:
Didn't realize it was equivalent, but yes, that seems a lot better :) Changed.